### PR TITLE
Day5 hw

### PR DESCRIPTION
**Pull request by OnkurLal**
updated to arrow function and added parameter to function.

### CODE REVIEW

File: js/style.js [hunk 1]
```diff
@@ -270,13 +270,13 @@
 
 // }
 
-async function pokemon(){
+const getPokemonData = async (pokemon) => {
```
Comment (victorSauceda) [inline js/style.js:273]: Nice. I liked how you passed in the `pokemon` parameter to this function to make it dynamic

File: js/style.js [hunk 2]
```diff
@@ -270,13 +270,13 @@
...
     try{
-        const getPokemon = await fetch('https://pokeapi.co/api/v2/pokemon/charizard');
+        const getPokemon = await fetch(`https://pokeapi.co/api/v2/pokemon/${pokemon}`);
         const pokemonData = await getPokemon.json();
-        alert(pokemonData.name)
+        alert(`The name of the pokemon for this data set is ${pokemonData.name}.`)
     }catch (err){
-        alert('There was an error');
+        alert('There was an error or the pokemon name was mispelled.');
```
Comment (victorSauceda) [inline js/style.js:279]: are we doing anything with the `err` parameter

**Review by victorSauceda — COMMENTED** [review]
Looks good. Always love how you go above and beyond

File: js/style.js (hunk 2 above)
Comment (OnkurLal) [reply js/style.js:279]: no, I'll remove it.

**Pull request merged**